upsertGitHubTag/deployment/index.js

```diff
@@ -253,6 +254,23 @@ function processEvent(event, callback) {
         " from GitHub.",
     });
   }
+  // If bucket name is not null (had to put this for the integration test)
+  if (process.env.BUCKET_NAME) {
```

Comment:
This is called after we invoke `callback` above...should it be before? Doesn't the lambda use `callback` to return a result? (general question, callbacks confuse me)

Reply:
It looks like it will still execute after the callback: https://stackoverflow.com/questions/49688927/how-do-i-stop-execution-of-a-aws-lambda-after-a-callback. Although maybe you want to log earlier to avoid confusion. It doesn't look like it will affect performance either way

Reply:
In looking at Kathy's question, I noticed the method is getting pretty big (we don't have a linter in place), so I'd optionally suggest creating a method out of this if block, e.g., `logPayloadToS3()`.